proto/stride/stakeibc/host_zone.proto

```diff
@@ -7,6 +7,19 @@ import "cosmos_proto/cosmos.proto";
 
 option go_package = "github.com/Stride-Labs/stride/v19/x/stakeibc/types";
 
+// CommunityPoolRebate stores the size of the community pool liquid stake
+// and the rebate percentage
+message CommunityPoolRebate {
```

Comment:
Might not have time, but it seems like we could generalize this by
* calling the struct `Rebate`
* storing the address on the struct
* making `community_pool_rebate` on the host zone (whatever it would be renamed to) a repeated field

Then this could support arbitrary DAOs, or chains with multiple community pools. Not needed for DYDX

Reply:
discussed this this morning - decided that it makes more sense to focus on just 1 pool for now since a second pool would require a different ICA message and DYDX only needs the main pool to be funded atm